src/applications/gi/components/vet-tec/VetTecContactInformation.jsx

```diff
@@ -117,13 +117,22 @@ export const VetTecContactInformation = ({ institution }) => {
     primarySCOs.length > 0 && (
       <div className="vads-l-row vads-u-margin-y--4">
         <div className="vads-l-col--12 medium-screen:vads-l-col--3">
-          <h4 className="contact-heading">Primary</h4>
+          <h4
+            id="primary-contact-header"
+            className="contact-heading"
+            aria-label="primary contact officials"
```

Comment:
This `aria-label` is not needed and could cause confusion for AT users. Let's remove it.